tests/integration/nm/ovs_test.py

```diff
@@ -85,7 +85,6 @@ def test_bridge_with_system_port(eth1_up, bridge_default_config):
     assert not _get_bridge_current_state()
 
 
-@pytest.mark.xfail(reason="https://github.com/nmstate/nmstate/issues/802")
```

Comment:
As there is not NM release with this fix yet, I guess we cannot remove this yet.